app-cdr/dvdisaster/dvdisaster-0.79.5.ebuild

```diff
@@ -20,36 +21,39 @@ done
 unset dvdi_lang
 
 RDEPEND="
-	>=dev-libs/glib-2.32:2
-	nls? ( virtual/libintl )
-	>=x11-libs/gtk+-2.6:2
+	dev-libs/glib:2
 	x11-libs/gdk-pixbuf
-"
-DEPEND="${RDEPEND}
+	x11-libs/gtk+:2
+	nls? ( virtual/libintl )"
+DEPEND="${RDEPEND}"
+BDEPEND="
 	nls? ( sys-devel/gettext )
 	virtual/os-headers
-	virtual/pkgconfig
-"
+	virtual/pkgconfig"
+
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
+
+DOCS=( CHANGELOG CREDITS.en README README.MODIFYING TODO TRANSLATION.HOWTO )
 
 src_configure() {
 	./configure \
-		--prefix=/usr \
-		--bindir=/usr/bin \
-		--mandir=/usr/share/man \
-		--docdir=/usr/share/doc \
+		--prefix="${EPREFIX}"/usr \
+		--bindir="${EPREFIX}"/usr/bin \
+		--mandir="${EPREFIX}"/usr/share/man \
+		--docdir="${EPREFIX}"/usr/share/doc \
 		--docsubdir=${PF} \
-		--localedir=/usr/share/locale \
-		--buildroot="${D}" \
-		--with-nls=$(usex nls) \
-		--with-memdebug=$(usex debug) || die
+		--localedir="${EPREFIX}"/usr/share/locale \
+		--buildroot="${ED}" \
```

Comment:
this is probably wrong, because it leads to a double prefix. change back to `${D}`

Reply:
Done